submissions/Rybchynskyi/app-Popup/css/style.css

```diff
@@ -0,0 +1,259 @@
+body {
```

Comment:
Please, check all this file and format it by one style: add 1 empty line after each block, start each rule from new line and remove necessary comments